### DISCUSSION TITLE
Network requests failed on Auth

### DISCUSSION THREAD

**Discussion by codeibrahima** [supabase/supabase#3671] [Questions]
This morning, I'm getting **Network requests failed** when trying to log in. 

No changes were made to the code, and It was working yesterday. 

I'm using Supabase Js client. 

Could someone help? 

Thank you.

**Comment by J0** [comment]
Hey @codeibrahima

It's been quite a while but I wanted to check in and see if you're still facing similar issues and if so what error messages you're getting under the Network Inspector tab.

Let us know!
